src/main/cpp/t06_longest.cpp

```diff
@@ -23,4 +23,18 @@ using namespace std;
 
 int t06_longest() {
 
+    string s, word, max;
+    getline(cin, s);
+
+    for (int i = 0; i <= s.size(); i++){
+        if (s[i] != ' ' && i != s.size() )
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When i==s.size(), s[i] is out of bounds.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4520560)